composer.json

```diff
@@ -136,7 +136,6 @@
         "View output is not used for autocomplete display (2174633)": "https://www.drupal.org/files/issues/2174633-198.patch",
         "Use new Transliteration functionality in core for file names (2492171)": "https://www.drupal.org/files/issues/use_new_transliteration-2492171-51.patch",
         "Move permission \"view any unpublished content\" from content moderation to core (273595)": "https://www.drupal.org/files/issues/move_permission_view-273595-129.patch",
-        "NegotiationMiddleware calls $request->setRequestFormat('html') when there is no _format request parameter, but shouldn't (2854543)": "https://www.drupal.org/files/issues/2018-04-23/2854543-72-PASS--51_plus_test_plus_67.patch",
```

Comment:
This isn't related but it's bugging me. The patch doesn't apply because it made it in to Drupal `8.5.6`.